src/vega_renderer/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project( vega_renderer )
 
-if(APPLE AND NOT TC_BUILD_IOS)
+if(APPLE)
```

Comment:
This is not the right change, I think... vega_renderer does not currently build on iOS.

Reply:
Fixed.